adapters/web/adapter-admin-api/src/test/java/com/sweet/user/controller/SignUpControllerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.sweet.user.controller;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.sweet.common.BaseControllerTest;
+import com.sweet.user.port.in.SignUpUseCase;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+@DisplayName("SignUpController 클래스")
+class SignUpControllerTest extends BaseControllerTest {
+
+  @MockBean
+  private SignUpUseCase signUpUseCase;
+
+  @Test
+  void testExample() throws Exception {
+    this.mockMvc.perform(post("/v1/com.sweet.user/sign-up")
```

Comment:
```suggestion
    this.mockMvc.perform(post("/v1/user/sign-up")
```

작업을 회사놋북에서 한뒤 git에 push를 할수 없어 파일 이동 이후 push하다가 잘못 수정이 되었네요.

용진님이 말씀하신 URL이 맞습니다.